File: Front/CampEventos-App/src/app/models/Evento.ts
import { Apresentador } from "./Apresentador";
import { Lote } from "./Lote";
import { RedeSocial } from "./RedeSocial";


export interface Evento{ 

    id : number;
    local : string;
    dataEvento : string;
    tema : string;
    qtdPessoas : number;
    imagemURL : string;
    telefone : string;
    email : string;
    descricao: string;
    modalidade: string;
    lotes : Lote[];
    redesSociais  : RedeSocial[];
    apresentadoresEventos : Apresentador[];
}

File: Back/src/CampEventos.Domain/Evento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using CampEventos.Domain.Identity;

namespace CampEventos.Domain
{
    public class Evento
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime? DataEvento { get; set; }
        public string Tema { get; set; }
        public int QtdPessoas { get; set; }
        public string ImagemURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public string Descricao { get; set; }
        public string Modalidade { get; set; }
        public int UserId {get; set;}
        public User USer { get; set; }
        public IEnumerable<Lote> Lotes { get; set; }
        public IEnumerable<RedeSocial> RedesSociais  { get; set; }
        public IEnumerable<ApresentadorEvento> ApresentadoresEventos { get; set; }
    }
}